bpf/lib/nodeport.h

```diff
@@ -87,6 +87,7 @@ struct dsr_opt_v6 {
 	__u8 opt_len;
 	union v6addr addr;
 	__be16 port;
+	__u16 pad;
```

Comment:
Do we also need to revert back to the old DSR_IPV6_OPT_LEN / DSR_IPV6_EXT_LEN ?

Reply:
I think there is no need to revert the definitions:

```
-#define DSR_IPV6_OPT_LEN       0x14    /* to store ipv6 addr + port */
-#define DSR_IPV6_EXT_LEN       0x2     /* = (sizeof(dsr_opt_v6) - 8) / 8 */
+#define DSR_IPV6_OPT_LEN       (sizeof(struct dsr_opt_v6) - 4)
+#define DSR_IPV6_EXT_LEN       ((sizeof(struct dsr_opt_v6) - 8) / 8)
```

`sizeof(struct dsr_opt_v6) == 24`, so both lens are the same.